aiida/manage/manager.py

```diff
@@ -33,6 +32,16 @@ class Manager:
       * reset manager cache when loading a new profile
     """
 
+    @staticmethod
+    def get_config():
```

Comment:
Since `get_config` is used in many places in the code, I'd just like to understand whether this is now the one we suggest for users.

Reply:
I actually just wrote this as a utility function for the `Manager` class itself and did not really think about it being used as the main way of retrieving the config for users. That is currently `from aiida.manage.configuration import get_config`. You'll notice that we haven't currently exposed it on `aiida.manage` level because I had to rewrite the config/profile code for `v1.0.0` and didn't know how stable/safe it was for random user use.

Reply:
Ok.